jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/FromSelectorTest.java

```diff
@@ -31,13 +31,13 @@
 
 public class FromSelectorTest {
 
-  @Mocked
+  @Mock
   JavaProject project;
 
-  @Mocked
+  @Mock
   Plugin plugin;
 
-  @Mocked
+  @Mock
```

Comment:
Considering that you didn't add the `@RunWith(MockitoJUnitRunner.class)` annotation and that the test is still passing, we can assume that you can safely remove these variables.